Dockerfile

```diff
@@ -8,17 +10,22 @@ RUN dnf install -y \
     && dnf config-manager --set-enabled powertools \
     && dnf clean all && rm -rf /var/cache/dnf/*
 RUN dnf install -y \
-        R R-devel R-Rcpp R-Rcpp-devel gdal gdal-devel \
```

Comment:
Was it intentional to get rid of `gdal` and `gdal-devel`? I think they are needed to build some R packages.